modules/openapi-generator/src/test/resources/3_0/oneoOfDiscriminator.yaml

```diff
@@ -0,0 +1,201 @@
+openapi: 3.0.1
+info:
+   title: fruity
+   version: 0.0.1
+paths:
+   /:
+      get:
+         responses:
+            '200':
+               description: desc
+               content:
+                  application/json:
+                     schema:
+                        $ref: '#/components/schemas/FruitAllOfDisc'
+components:
+   schemas:
+    FruitType:
+      properties:
+        fruitType:
+          type: string
+      required:
+      - fruitType
+    FruitInlineDisc:
+      oneOf:
+        - type: object
+          required:
+            - seeds
+            - fruitType
+          properties:
+            seeds:
+              type: integer
+            fruitType:
+              type: string
+        - type: object
+          required:
+            - length
+            - fruitType
+          properties:
+            length:
+              type: integer
+            fruitType:
+              type: string
+      discriminator:
+        propertyName: fruitType
+    FruitInlineInlineDisc:
+      oneOf:
+        - type: object
```

Comment:
These inline models should be ignored per spec

Reply:
These ones I can detect because the inline model resolver does not extract them and we see the null $refs

Reply:
As commented elsewhere, that's something we'll need to change to be spec compliant but not related to your PR.